lib/upgradable-multisig/multisig/validation.ak

```diff
@@ -2,21 +2,23 @@ use aiken/collection/dict
 use aiken/collection/list
 use aiken_design_patterns/singular_utxo_indexer
 use cardano/address.{Address, Script}
-use cardano/assets.{PolicyId, Value, quantity_of, tokens}
+use cardano/assets.{PolicyId, quantity_of, tokens}
 use cardano/transaction.{
   InlineDatum, Input, Output, OutputReference, Transaction,
 }
 use upgradable_multisig/multisig/utils.{
-  multisig_token_name, signed_within_threshold, validate_multisig_datum,
+  signed_within_threshold, validate_multisig_datum,
 }
 use upgradable_multisig/types.{MultisigDatum}
 
-pub fn validate_initiate_multisig(
+pub fn validate_init(
   own_policy: PolicyId,
+  own_ref: OutputReference,
```

Comment:
It is not checked that the input with `own_ref` is spent! Also, please see comments for types.ak.